src/Wallabag/CoreBundle/Filter/EntryFilterType.php

```diff
@@ -5,12 +5,15 @@
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Lexik\Bundle\FormFilterBundle\Filter\FilterOperands;
 
 class EntryFilterType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('readingTime', 'filter_number_range');
+        $builder
+            ->add('readingTime', 'filter_number_range')
+            ->add('domainName', 'filter_text', array('condition_pattern' => FilterOperands::STRING_BOTH));
```

Comment:
Could you add a validator to avoid people to search for `a` and kill the database ?I guess a minimum value of 3 characters should be ok.